src/content/recipes/sass.md

```diff
@@ -0,0 +1,60 @@
+<div class="aside aside__no-top">
+
+Some configurations of Storybook already come pre-configured to support sass. If your project meets the following, you're likely ready to go.
```

Comment:
I can't tell if it's supposed to be `sass` or `Sass`, but we should try to be consistent.

Reply:
Thanks Kyle 😀 Updated